ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,250 @@
+
+# -p 79A3:2015.6 799C:2102.9 7986:2513.2
+# -ii 7FB7 7FB5
+# -it "Cagnazzo"
+
+2000.0 "--sync--" sync / 00:0839::Deepspine will be sealed off/ window 2000,1
+2015.6 "Stygian Deluge" sync / 1[56]:[^:]*:Cagnazzo:79A3:/
+2025.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:798F:/
+2038.0 "Antediluvian 1" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2042.0 "Antediluvian 2" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2043.3 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7991:/
+2044.3 "Body Slam" sync / 1[56]:[^:]*:Cagnazzo:(7992|7993):/
+2046.0 "Antediluvian 3" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2051.1 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2060.9 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7FB4:/
+2062.8 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2063.2 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2063.9 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2064.3 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2065.0 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2065.4 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2066.1 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2066.5 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2067.2 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2067.6 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2068.3 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2068.7 "Hydrobomb x2" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2069.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7996:/
+2073.5 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2084.6 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7A8F:/
+2085.2 "Hydrofall" sync / 1[56]:[^:]*:Cagnazzo:7A90:/
+2095.9 "Cursed Tide" sync / 1[56]:[^:]*:Cagnazzo:799A:/
+
+# add phase
+2099.5 "--untargetable--"
+2099.5 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2102.9 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799C:/
+2113.7 "Neap Tide" sync / 1[56]:[^:]*:Cagnazzo:799E:/
+2118.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+2124.7 "Spring Tide" sync / 1[56]:[^:]*:Cagnazzo:799F:/
+2134.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+2145.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+# probably more Hydrovent spam here until phase end or enrage
+2146.4 "--sync--" sync / 00:0044:Cagnazzo:No more games!/ window 30,30
+2159.1 "Tsunami" sync / 1[56]:[^:]*:Cagnazzo:79A1:/ window 60,60
+
+2500.0 "--targetable--"
+2506.1 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2513.2 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7986:/
+2514.1 "Voidcleaver" sync / 1[56]:[^:]*:Cagnazzo:7987:/
+2523.1 "Void Miasma 1" sync / 1[56]:[^:]*:Cagnazzo:7FB3:/
+2529.1 "Void Miasma 2" sync / 1[56]:[^:]*:Cagnazzo:7FB3:/
+2529.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7988:/
+2530.4 "Lifescleaver x8" sync / 1[56]:[^:]*:Cagnazzo:7989:/
+2535.1 "Void Miasma 3" sync / 1[56]:[^:]*:Cagnazzo:7FB3:/
+2539.8 "Void Torrent" sync / 1[56]:[^:]*:Cagnazzo:798E:/
+2547.6 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2557.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7FB4:/
+2559.3 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
```

Comment:
Sorry, I thought I had left a comment here, but maybe it got lost?  This is a lot of lines in close proximity that I think make the timeline harder to read.  Can this (and the section below) be a single Hydraulic Ram with a duration to cover everything else?  Maybe hydrobomb can be skipped.